build.psm1

```diff
@@ -2386,6 +2394,7 @@ function Start-CrossGen {
     $crossGenPath = Get-ChildItem $script:Environment.nugetPackagesRoot $crossGenExe -Recurse | `
                         Where-Object { $_.FullName -match $crossGenRuntime } | `
                         Where-Object { $_.FullName -match $dotnetRuntimeVersion } | `
+                        Where-Object { (Split-Path $_.FullName -Parent).EndsWith('tools') } | `
```

Comment:
Is this 'tools' path filter just for macOS, or for all runtimes?  I assume the latter, but it is not clear to me.  Maybe add a comment to reassure.

Reply:
The `tools` folder is for all runtimes, but for some reason, x64-arm64 for macOS has a `crossgen` in that additional runtime folder which is discovered, but doesn't actually work.  The one in `tools` does work.